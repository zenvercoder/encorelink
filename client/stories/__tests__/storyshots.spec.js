import initStoryshots from 'storyshots';

initStoryshots({
  configPath: 'client/stories'
});
