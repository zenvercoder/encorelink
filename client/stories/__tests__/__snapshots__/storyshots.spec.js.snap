exports[`Storyshots Header Logged In: Organizer 1`] = `
<header
  className="top-bar">
  <div
    className="top-bar-left">
    <a
      onClick={[Function]}
      style={Object {}}>
      <img
        alt="EncoreLink"
        className="logo-img"
        src="/public/img/encorelink-logo.png" />
    </a>
  </div>
  <div
    className="top-bar-right text-right">
    <div>
      Hello, 
      <a
        onClick={[Function]}
        style={Object {}}>
        organizer@encorelink.com
      </a>
       
      <a
        className="button"
        onClick={[Function]}
        style={Object {}}>
        Dashboard
      </a>
       
      <button
        className="button"
        onClick={[Function]}>
        Log out
      </button>
    </div>
  </div>
</header>
`;

exports[`Storyshots Header Logged in: Musician 1`] = `
<header
  className="top-bar">
  <div
    className="top-bar-left">
    <a
      onClick={[Function]}
      style={Object {}}>
      <img
        alt="EncoreLink"
        className="logo-img"
        src="/public/img/encorelink-logo.png" />
    </a>
  </div>
  <div
    className="top-bar-right text-right">
    <div>
      Hello, 
      <a
        onClick={[Function]}
        style={Object {}}>
        musican@encorelink.com
      </a>
       
      <a
        className="button"
        onClick={[Function]}
        style={Object {}}>
        Dashboard
      </a>
       
      <button
        className="button"
        onClick={[Function]}>
        Log out
      </button>
    </div>
  </div>
</header>
`;

exports[`Storyshots Header Not logged in 1`] = `
<header
  className="top-bar">
  <div
    className="top-bar-left">
    <a
      onClick={[Function]}
      style={Object {}}>
      <img
        alt="EncoreLink"
        className="logo-img"
        src="/public/img/encorelink-logo.png" />
    </a>
  </div>
  <div
    className="top-bar-right text-right">
    <a
      className="button"
      onClick={[Function]}
      style={Object {}}>
      Log in
    </a>
  </div>
</header>
`;
